SQF/dayz_code/stringtable.xml

```diff
@@ -15763,9 +15763,13 @@
 			<English>No info found</English>
 			<Russian>Нет информации</Russian>
 		</Key>
-		<Key ID="STR_EPOCH_TRADE_NO_ITEMS">
-			<English>No items to sell</English>
-			<Russian>Нет ничего на продажу</Russian>
+		<Key ID="STR_EPOCH_TRADE_BUY_NO_ITEMS">
+			<English>No items in buy list</English>
+			<Russian>???? ???? ???? ????</Russian>
+		</Key>
+		<Key ID="STR_EPOCH_TRADE_SELL_NO_ITEMS">
```

Comment:
I'm not sure why you changed this translation. "No items sell list" does not make sense in English. "No items to sell" was correct.